test/e2e/density.go

```diff
@@ -361,18 +367,29 @@ var _ = framework.KubeDescribe("Density", func() {
 		{podsPerNode: 50, runLatencyTest: false},
 		{podsPerNode: 95, runLatencyTest: true},
 		{podsPerNode: 100, runLatencyTest: false},
+		{podsPerNode: 30, runLatencyTest: true, kind: extensions.Kind("Deployment")},
```

Comment:
OK. But let's treat this PR as purely infrastructural, and let's add this in a separate PR.